src/components/RecipeEdit.vue

```diff
@@ -331,12 +362,17 @@ export default {
         this.$root.$off('saveRecipe')
         this.$root.$on('saveRecipe', () => {
             this.save()
+            this.savingRecipe = true
```

Comment:
This will set the saving flag after the saving has been already started/triggered. Maybe we should set it before.

Reply:
Yes, I think it’s a good idea to set it in the save function. I’ll update the PR